owlkettle/widgets.nim

```diff
@@ -3401,6 +3401,109 @@ renderable AboutDialog of BaseWidget:
         "Art": @["Max Mustermann"]
       }
 
+type ScalePosition* = enum
+  ScaleLeft,
+  ScaleRight,
+  ScaleTop,
+  ScaleBottom
+
+proc toGtk(pos: ScalePosition): GtkPositionType =
+  result = GtkPositionType(ord(pos))
+
+type Mark* = tuple
```

Comment:
`Mark` is a very generic name (especially since it is exported). Maybe `ScaleMark`?

Reply:
Done